code/style.css

```diff
@@ -1,4 +1,379 @@
+    width: 90%;
+    padding: 14px;
+    box-sizing: border-box;  
+    margin: 20px auto;  
+    box-shadow: 0 4px 8px 0 rgba(0, 0, 0, 0.2), 0 6px 20px 0 rgba(0, 0, 0, 0.19);
+}
+
+.other-article-child img {
+    width: 100%;
+    box-shadow: 7px 7px gainsboro;
+}
+
+.other-article-child h2 {
+    border-bottom: solid 1px #214273;
+    padding: 15px 0px 10px 0px;
+    margin: 0;
+    font-size: 20px;
+    text-align: center;
+}
+
+.other-article-child h2 a {
+    color: #214273;
+    text-decoration: none;
+}
+
+.other-article-child h2 a:hover {
+    color: #fa5aaa;
+}
+
+.other-article-child p {
+   background: #ffd7de;
+   padding: 10px 15px;
+}
+
+.read-more a{
+    background: #ff8cc5;
+    text-decoration: none;
+    border-radius: 2px;
+    font-family: 'Montserrat', sans-serif; 
+    font-size: 10px;
+    font-weight: 600;
+    color:  white;
+    margin: 0 0;
+    padding: 5px;
+    box-sizing: border-box;
+}
+
+.read-more a:hover {
+    background: #fda7d2;
+    color: white;
+} 
+
+/* FOOTER */
+
+.footer-title {
+    display: flex;
+    flex-direction: column;
+    align-items: center;
+    justify-content: center;
+    margin-bottom: 10px;
+    box-sizing: border-box;
+}
+
+.footer-title h1 {
+    font-size: 23px;
+    padding-bottom: 5px;
+    margin-bottom: 10px;
+    border-bottom: solid 1px #214273;
+    box-sizing: border-box;
+}
+
+.footer-title h5 {
+    padding: 2px;
+    margin: 10px 10px;
+}
+
+.footer-title img {
+    width: 40px;
+}
+
+.footer-header-container img{
+    margin-top: 0px;
+    margin-bottom: 10px;
+} 
+
+.sm-icons-container {
+    color: rgb(255, 140, 197);
+    display: flex;
+    flex-direction: row;
+    justify-content: space-evenly;
+    align-items: center;
+    flex-wrap: wrap;
+    margin: 10px;
+    box-sizing: border-box;
+}
+
+.sm-icons-container a {
+    color: #214273;
+    margin-right: 20px;
+    margin-left: 20px; 
+    font-size: 30px;
+}
+
+.sm-icons-container a:hover {
+    color:#fda7d2;
+} 
+
+/* MEDIA QUERY 1: SCREEN SIZE 768PX OR BIGGER & 2 CARDS PER ROW*/
+
+@media (min-width:768px) {
+
+    .hamburger-menu {
+        display: none;
+    }
+
+    .header-parent nav {
+        display: flex;
+        justify-content:space-between;
+        align-content: center;
+    }
+
+    .header-parent nav a {
+        text-decoration: none;
+        font-family: 'Montserrat', sans-serif;
+        color: #214273;
+        padding: 0 15px;
+        font-size: 16px;
+    }
+
+    .header-parent nav a:hover {
+       color: #ff8cc5;
+    }  
+
+    .main-article-child {
+        flex-direction: row;
+    }
+
+    .main-article-child img {
+        width: 50%;
+    }
+
+    .text-area {
+        padding: 10px 15px;
+        width: 500px;
+        background: white;
+        box-sizing: border-box;
+        margin: 0px 0px 0px 5px;
+        display: flex;
+        flex-direction: row;
+        flex-wrap: wrap;
+        align-items: center;
+        justify-content: center;
+        margin: 0 10px;
```

Comment:
For the text-area class you seem to have two margin values, it will always the last one written that will be displayed.